winlogbeat/eventlog/wineventlog.go

```diff
@@ -664,7 +664,7 @@ func newInputMetrics(name, id string) *inputMetrics {
 	if id == "" {
 		return nil
 	}
-	reg, unreg := inputmon.NewInputRegistry(name, id, nil)
+	reg, unreg := inputmon.NewInputRegistry("winlog", id, nil)
```

Comment:
Could we differentiate winlog streams from different providers? or is that reasonably captured by "name"?

Reply:
We'll still have the "provider" value as a string metric. We may want to know about the `forwarded` attribute that can be set so that might be a future addition (but we'll probably be looking at config when analyzing the data).

(Also I'm regretting using `name` instead of `type` in the data exposed through the `/inputs/` API.)